StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheet.swift

```diff
@@ -164,10 +164,17 @@ public class CustomerSheet {
 extension CustomerSheet {
     func loadPaymentMethodInfo(completion: @escaping (Result<([STPPaymentMethod], CustomerPaymentOption?, STPElementsSession), Error>) -> Void) {
         Task {
+            if let paymentMethodTypes = self.customerAdapter.paymentMethodTypes {
+                if case .failure(let error) = paymentMethodTypes.customerSheetSupportedPaymentMethodTypes() {
+                    completion(.failure(error))
+                    return
+                }
+            }
```

Comment:
Do we need to call this here?  It's already going to be called by `fetchPaymentMethods`

Reply:
oops! thank you for catching this! should have been deleted but wasn't!